app/services/pdf_service.rb

```diff
@@ -10,6 +10,8 @@ class PdfService
     emulate_media: "screen",
     launch_args: ["--font-render-hinting=medium", "--no-sandbox"],
     execute_script: "document.querySelectorAll('button').forEach(el => el.style.display = 'none')",
+    media_features: [{ name: "prefers-color-scheme", value: "dark" }],
```

Comment:
Is this related to the fix or does it have some other effect?

Reply:
using just `scale:` changes the scale but not the splitting of the title into multiple blocks (even at 0.5/1 where the text fits comfortably on the line). adding `media_features` prevents the title being split into multiple sections when the text will fit on a single line, or if it doesn't fir on a single line, each line is its own block (as opposed to the current situation where it is chunked up and split mid-word)